_posts/phuong-my_do/2017-04-28-Business_analyst.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: post
+title: "Tea Chat with a Business Analyst (BA)"
+breadcrumb: true
+author: phuong-my_do
+date: 2017-06-19
+categories: phuong-my_do
+tags:
+    - business analyst
+teaser:
+    info: A Business Analyst working in Bioinformatics in a Software Engineering Team, what is that?
+---
+
+__What is a Business Analyst in the Cancer Genomics and Bioinformatics world?__
+
+If asking this question to employees in a Cancer Research Institute, chances are 90% of respondents would have no idea; 10% may have some ideas... and we don’t want to know how many of the 10% would be able to describe a Business Analyst job. :)
```

Comment:
It was a joke, implying that even though people would know what a BA is, they would not be able to describe the job. Like your grandma knows you're in IT but when describing it, she would say you repair computers.